apps/extension/src/ui/domains/Asset/TokenPicker.tsx

```diff
@@ -189,6 +190,7 @@ const TokensList: FC<TokensListProps> = ({
   const { evmNetworksMap } = useEvmNetworks(useTestnets)
   const { tokens: allTokens } = useTokens(useTestnets)
   const tokenRatesMap = useTokenRatesMap()
+  const formatNetworkName = useFormatNetworkName()
```

Comment:
having a hook returning just a callback feels weird, wdyt about making chain & evmNetwork arguments of the hook, and have the hook return the formatted label directly ?

Reply:
It's a valid pattern. The function needs to be used in places where you can't directly call a hook, like inside another hook in this component.